doc/REST-interface.md

```diff
@@ -52,6 +52,20 @@ With the /notxdetails/ option JSON response will only contain the transaction ha
 Given a block hash: returns <COUNT> amount of blockheaders in upward direction.
 Returns empty if the block doesn't exist or it isn't in the active chain.
 
+#### Blockfilter Headers
+`GET /rest/blockfilterheaders/<FILTERTYPE>/<COUNT>/<BLOCK-HASH>.<bin|hex|json>`
```

Comment:
@TheBlueMatt: Any thoughts on the above?

Reply:
I think the reason for this PR to use `GET /rest/blockfilterheaders/<FILTERTYPE>/<COUNT>/<BLOCK-HASH><bin|hex|json>`  was that the [**blockheader** endpoint](https://github.com/bitcoin/bitcoin/blob/master/doc/REST-interface.md#blockheaders) already uses the same format. (`GET /rest/headers/<COUNT>/<BLOCK-HASH>.<bin|hex|json>`)

@stickies-v makes a good point but in my opinion staying consistent with the current API was the better move.

Reply:
Thanks for taking a look @dergoegge . I was thinking about this and assumed that this was done for consistency as well.

Reply:
I agree that consistency is important, and in that light am happy to consider this comment as resolved on this PR. However, I think having just 1 prior case should not prevent us from changing course slightly and adopting better RESTful practices. I think this can nicely be done in a separate PR, so I'm happy to look into that and see if there's an elegant way to remain backwards compatible and still adopt/encourage best practices going forward, specifically on these `/headers/` and `/blockfilterheaders/` endpoints.